src/gc.c

```diff
@@ -785,21 +785,21 @@ static inline int maybe_collect(void)
 
 // preserved values
 
-DLLEXPORT int jl_gc_n_preserved_values(void)
+int jl_gc_n_preserved_values(void)
```

Comment:
OK, I'll put the DLLEXPORTs back on thos... I just don't like having unused stuff cluttering up an API...